.github/workflows/Release.yml

```diff
@@ -0,0 +1,82 @@
+name: "Publish Release"
+on:
+  workflow_dispatch:
+env:
+  build_dir: ./build
+  tarballs_dir: ./build/tarballs
+jobs:
+  release:
+    name: "Publish Pre-Release"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Needed determine latest tag
+      - name: Get Latest Tag
+        id: latest_tag
+        uses: WyriHaximus/github-action-get-previous-tag@v1
+        # TODO: in future we might need to perform backports, at which point we'll need to check
+        # that the project.version has not already been tagged 
```

Comment:
Just curious if you tried `git fetch --tags`? If that works then you can change the check later to `!(project.version in tags)` to support backports

Reply:
yeah I think it worked - I'll do it as a follow up. Afraid of accidentally breaking things now...